composer.json

```diff
@@ -15,10 +15,10 @@
     ],
     "require": {
         "php": "^7.1",
-        "illuminate/support": "5.5.* || 5.6.* || 5.7.* || 5.8.*"
+        "illuminate/support": "^5.7 || ^6.0"
```

Comment:
I think `"^6.0"` would be better.

Then I just tag it as a "major" release that introduces breaking changes.

Reply:
Just a note, if the package is going to become Laravel 6.0 and above, I guess PHP needs to be updated to require `^7.2`, Travis needs to be updated to not run against `7.1`, and PHPUnit needs to be updated to `^8.0` (otherwise [this happens](https://travis-ci.com/pxgamer/laravel-sri/jobs/231031190)). 👍

Reply:
Good point.

I've been out of the Laravel (and PHP) world for a couple of months now, so I haven't really been able to keep up with what was happening with the releases 😂